documentation/general/eliminate-duplicate-file.md

```diff
@@ -120,102 +124,120 @@ Notice the 33% increase in duplicate file size in 10.0 compared to 9.0.
+
+**macOS Installer (pkg):**
+
+The macOS pkg installer format supports hard links and will preserve them during installation, maintaining disk space savings on macOS systems.
+
+**Windows Installer (MSI):**
+
+MSI installers do not natively support hard links.
+Supporting hard links in MSI would require custom logic that Windows installer subject matter experts have identified as problematic and do not recommend pursuing.
+Further investigation is need to confirm this.
+As a result, Windows MSI installations will likely not benefit from hard link deduplication and will consume additional disk space compared to tarball installations on Windows.
+The Windows installers at least have historically had a more 'maximal' approach to what is included in them, so there's precedence for the methods to have deltas.
 
 ### Testing Strategy
 
-Once duplicated assemblies are removed, a regression test should:
+Tests will be created to ensure:
 
-- Detect duplicate files in the SDK layout.
-- Fail the build if duplicates are found.
-- Prevent regressions.
+- **Layout validation:** Verify that duplicate files outside of hardlinks do not exist
+- **Archive Extraction testing:** Verify that tarball extraction preserves hardlinks on all supported platforms
+- **Installer testing:** Verify the non-windows installers preserver hardlinks
 
 ## Proof of Concept Results
 
-A proof of concept was implemented for `dotnet-watch` and `dotnet-format`, two of the largest sources of duplication.
-The POC used the **AssemblyLoadContext** approach with a shared assembly location and yielded the following results:
+A proof of concept for hard link deduplication was implemented, demonstrating the viability and effectiveness of the approach:
+
+**Implementation Details:**
+- A new MSBuild task `DeduplicateFilesWithHardLinks` was created to identify and replace duplicate files with hard links during SDK layout generation
+- Files are hashed using SHA256 to identify duplicates
+
+**Space Savings:**
 
-| Metric       | Baseline 11.0 (SDK only) | Size Reduction | % Reduction |
-|--------------|--------------------------|----------------|-------------|
-| Archive Size | 100 MB                   | 21 MB          | 21.0%       |
-| Disk Size    | 296 MB                   | 62 MB          | 20.8%       |
-| File Count   | 3,957                    | 384 files      | 9.7%        |
+In a Linux x64 development build of .NET 11.0 SDK, the hard link approach achieved:
+- 131 MB reduction in disk size
+- 61 MB reduction in archive size
 
-**Note:** These numbers are from a Linux development build, which differs significantly from official signed/optimized builds included in the [By the Numbers](#by-the-numbers) section.
+**Note:** Development build measurements differ from production builds due to signing, optimization, and release packaging differences.
+Production measurements will be captured during full implementation.
```

Comment:
One thing to validate is sign tool, which re-packs the archive to sign binaries. Need to make sure it preserves hard links.